doc/advanced/CMakeLists.txt

```diff
@@ -3,7 +3,12 @@ if(NOT SPHINX_FOUND)
 endif()
 
 add_custom_target(advanced ALL
-                  COMMAND "${SPHINX_EXECUTABLE}" -b html -a -d "${SPHINX_CACHE_DIR}" -D html_file_suffix=".${SPHINX_HTML_FILE_SUFFIX}" "${CMAKE_CURRENT_SOURCE_DIR}/content" html)
+                  COMMAND "${SPHINX_EXECUTABLE}"
+                    -b html
+                    -d "${SPHINX_CACHE_DIR}"
+                    -Dversion="${PCL_VERSION_PRETTY}"
```

Comment:
Should not the version and release be different variables?

Reply:
And what do you suggest to put there?

Reply:
1. Manually supply `${PCL_VERSION_MAJOR}.${PCL_VERSION_MINOR}.${PCL_VERSION_PATCH}`
2. Create another variable (say `PCL_VERSION_NUMERICAL`) in `pcl_utils.cmake` along with `PCL_VERSION_PTETTY` and use that.

It's just that documentation says that version should be similar to semantic version, but pretty is definitely not semantically versioned

Reply:
Version is the variable used to render this
![Screenshot from 2020-05-04 11-30-35](https://user-images.githubusercontent.com/1709142/80952976-ea45ba00-8dfa-11ea-8827-3a37f1787a3a.png)
I believe it's more important for users to know if they're working with the development version or not
![Screenshot from 2020-05-04 11-27-39](https://user-images.githubusercontent.com/1709142/80952668-4eb44980-8dfa-11ea-8945-0269db12b9f2.png)

That way, when they checkout specific tags/version it will be semver compliant, but if they checkout anything else, the `-dev` will be there.

Plus, semver is not a strict requirement for version. RTD sets it to `latest` and `stable` for instance.